src/app.md

```diff
@@ -119,7 +119,22 @@ Your favorite number must be 256.
 
 [![log-badge]][log] [![env_logger-badge]][env_logger] [![cat-debugging-badge]][cat-debugging]
 
-[Write me!](https://github.com/brson/rust-cookbook/issues/61)
+```rust
+#[macro_use] extern crate log;
+extern crate env_logger;
+
+fn main() {
+    env_logger::init().unwrap();
+
+    error!("this is an error {}", "message");
+}
+```
+
+Run this code with `cargo run` and you should see the following line:
```

Comment:
Imho the debug example could describe basics while this one would assume that reader is already acquainted with the previous one and would focus on the part that is different between them.